addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16870,7 +16870,7 @@ msgstr ""
 #. Description of setting with label #13510 "Sync playback to display"
 #: system/settings/settings.xml
 msgctxt "#36166"
-msgid "Synchronise video and audio to the refresh rate of the monitor. VideoPlayer won't use passthrough audio in this case because resampling may be required."
+msgid "Synchronise video and audio to the refresh rate of the connected display. Since this function may require the resampling of audio, when this setting is enabled the \"Allow passthrough\" audio setting is disabled."
```

Comment:
That sentence is hard to read and hard to parse :-). Syncing to display will disable passthrough. On cannot run clocked with the Display and passing through Audio.